feature/settings/src/main/java/com/serranoie/app/feature/settings/trip/TripSettingsScreen.kt

```
@@ -42,82 +46,98 @@ import androidx.compose.runtime.remember
 import com.serranoie.app.designsystem.ui.theme.component.IButton
-import com.serranoie.app.designsystem.ui.theme.component.OutlinedCard
+import com.serranoie.app.designsystem.ui.theme.component.OtpDisplayField
+import com.serranoie.app.designsystem.ui.theme.component.card.ICard
+import com.serranoie.app.designsystem.ui.theme.component.PaddedListGroup
+import com.serranoie.app.designsystem.ui.theme.component.PaddedListItemPosition
+import com.serranoie.app.itinero.feature.settings.trip.TripSettingsViewModel
 
-@OptIn(ExperimentalMaterial3Api::class)
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalMaterial3ExpressiveApi::class)
 @Composable
 fun TripSettingsScreen(
```

detekt (1.23.8)

[warning] The function TripSettingsScreen appears to be too complex based on Cognitive Complexity (complexity: 26). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)